controllers/admin/chat.go

```diff
@@ -58,3 +55,57 @@ func GetChatMessages(w http.ResponseWriter, r *http.Request) {
 		log.Errorln(err)
 	}
 }
+
+// SendSystemMessage will send an official "SYSTEM" message
+// to chat on behalf of your server.
+func SendSystemMessage(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "application/json")
+
+	var message models.ChatEvent
+	if err := json.NewDecoder(r.Body).Decode(&message); err != nil {
+		controllers.InternalErrorHandler(w, err)
+		return
+	}
+
+	message.MessageType = "SYSTEM"
```

Comment:
It's in my PR, but just calling this out as one we will need to adjust when merging my PR down since this is MessageType now